modules/ROOT/pages/policy-extras.adoc

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
Not edited as this is all commented out.

Reply:
Yes, we don't need this content for now.